test/unit/domains.server.test.ts

```diff
@@ -0,0 +1,221 @@
+import { createHostedZone } from '~/lib/dns.server';
+import {
+  createUserDomain,
+  setMonthsFromNow,
+  deleteUserDomain,
+  willExpireIn,
+  isExpired,
+  removeIfExpired,
+  updateUserDomain,
+} from '~/lib/domains.server';
+import { prisma } from '~/db.server';
+import { RecordType } from '@prisma/client';
+
+import type { Domain } from '../../app/lib/domains.server';
```

Comment:
`from '~/lib/domains.server';`

Reply:
It showed me an error when I use `~` before. Will be updated.